src/coprocessor/endpoint.rs

```diff
@@ -97,22 +95,20 @@ impl CopContext {
 
 impl Context for CopContext {
     fn on_tick(&mut self) {
-        if self.task_count == 0 {
-            return;
-        }
-        let task_count = self.task_count;
-        for type_str in &[STR_REQ_TYPE_SELECT, STR_REQ_TYPE_INDEX, STR_REQ_TYPE_DAG] {
+        for type_str in &[STR_REQ_TYPE_SELECT, STR_REQ_TYPE_INDEX] {
             let this_statistics = self.get_statistics(type_str);
-            for (cf, details) in this_statistics.details() {
+            if this_statistics.count == 0 {
+                continue;
+            }
+            for (cf, details) in this_statistics.stat.details() {
                 for (tag, count) in details {
                     COPR_SCAN_DETAILS
                         .with_label_values(&[type_str, cf, tag])
-                        .observe(count as f64 / task_count as f64);
+                        .observe(count as f64 / this_statistics.count as f64);
```

Comment:
I don't know exactly what is the metric COPR_SCAN_DETAILS used for, and why do we need to use divide the count here. Can you give me a metric example?

Reply:
We will implement `LocalHistogramVec` and use it instead later @siddontang

Reply:
Got it